streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -52,10 +53,8 @@ public void before() {
     }
 
     @After
-    public void cleanup() {
-        if (stateDir.exists()) {
-            Utils.delete(stateDir);
-        }
+    public void cleanup() throws IOException {
+        Utils.delete(stateDir);
```

Comment:
This `exists` is not necessary because `Utils#delete` does not fail when given a non-existent path.  (It had this behavior earlier as well, this patch didn't change it.)